README.md

```diff
@@ -10,7 +10,7 @@
 
 Nextcloud app to sign PDF documents.
 
-<img src="img/screenshot/request-signature.png" />
+<img src="img/LibreSign.png" />
```

Comment:
Is the opposite than this. The previous state is right.

You also need to remove the `img/LibreSign.png` file.